demo/util/confirm-features.mjs

```diff
@@ -0,0 +1,25 @@
+function redDiv(textContent) {
+  const div = document.createElement('div');
+  div.style.color = 'red';
+  div.textContent = textContent;
+  return div;
+}
+
+/**
+ * Checks that the features needed are present in the browser. If not,
+ * it places error messages inside |element|.
+
+ **/
+export function confirm(element) {
+  if (element.displayLock === null) {
```

Comment:
```suggestion
  if (element.displayLock === undefined) {
```

It will actually be undefined in browsers that don't support it, so the red div will not be displayed with the current code.

Reply:
done